src/util/glob.ts

```diff
@@ -131,8 +144,14 @@ export async function glob(
   // We want each group to include all subsequent negated patterns. The simplest
   // way to do that is to build the groups backwards.
   for (let i = expandedPatterns.length - 1; i >= 0; i--) {
-    const pattern = expandedPatterns[i];
+    let pattern = expandedPatterns[i];
     const isExclusive = pattern[0] === '!';
+    if (isExclusive) {
+      pattern = pattern.slice(1); // Remove the "!"
+    }
+    if (opts.rerootToCwd) {
```

Comment:
Should this be an option or enabled by default? All use cases (except tests) pass in `rerootToCwd = == true`.

Reply:
True. I've just made it the only behavior, since we want it everywhere.

PTAL